ansible/roles/ocp4-workload-debezium-demo/tasks/provision_webapp.yaml

```diff
@@ -17,42 +17,44 @@
 
 - name: Download example files
   unarchive:
-      src: '{{ocp4_workload_debezium_demo_webapp_operator_resources}}'
-      dest: '{{ webapp_operator_tmp }}'
-      remote_src: yes
+    src: '{{ ocp4_workload_debezium_demo_webapp_operator_resources}}'
+    dest: '{{ webapp_operator_tmp }}'
+    remote_src: true
 
 - name: Create WebApp Operator Resources
   k8s:
     state: present
     namespace: '{{ webapp_namespace }}'
-    src: '{{ webapp_operator_tmp }}/tutorial-web-app-operator-{{ ocp4_workload_debezium_demo_webapp_operator_release_tag }}/deploy/{{ item }}'
-  with_items: "{{ ocp4_workload_debezium_demo_webapp_operator_resource_items }}"
+    src: >-
+      {{ '{}/tutorial-web-app-operator-{}/deploy/{}'.format(
+        webapp_operator_tmp, ocp4_workload_debezium_demo_webapp_operator_release_tag, item
+      ) }}
+  loop: "{{ ocp4_workload_debezium_demo_webapp_operator_resource_items }}"
```

Comment:
i don't think using `.format()` is more clear here.

Reply:
Fair enough. It weighs in at 137 characters with just inline. Which seems not unreasonable. I'm looking to suggest 150 character line length cutoff in the future.